h/accounts/schemas.py

```diff
@@ -201,7 +201,6 @@ class EmailChangeSchema(CSRFSchema):
     password = password_node(title=_("Confirm password"), hide_until_form_active=True)
 
     def validator(self, node, value):
-        super().validator(node, value)
```

Comment:
The `CSRFSchema` superclass no longer has a `validate()` method, so several of these `super()` calls had to be removed. It'll no longer be possible to accidentally disable CSRF protection by forgetting to call `super()` here.